packages/e2e-test-utils-playwright/src/request/index.ts

```diff
@@ -122,6 +123,10 @@ class RequestUtils {
 	addWidgetBlock = addWidgetBlock;
 	deleteAllTemplates = deleteAllTemplates;
 	resetPreferences = resetPreferences;
+	listMedia = listMedia;
```

Comment:
Off-topic: I'm starting to think that maybe we should namespace these by file names. So `requestUtils.listMedia` becomes `requestUtils.media.list`.

Reply:
How are you thinking that would look in terms of the exports, imports?

Reply:
I don't think that would be much different? Either way it should be implementation detail and we usually don't care about tree-shaking for e2e test utils anyway.

I'm thinking maybe `media.js` can export a class and `requestUtils` can then initiate an instance.

```js
// request/media.js
export class MediaRequestUtils { /* ... */ };

// request/index.ts
media = new MediaRequestUtils( this );
```

Reply:
It seems like Classes can end up as superfluous wrappers in js modules, unless there is a reason to want to maintain state across methods in an OOP style way. Just exporting a single default object which includes all the util methods would be another option ... but I don't have a strong opinion either way.